### PR TITLE
Display Map From Mobile Map Package Sample

### PR DESCRIPTION
**Pull request by shokhinaToj**
Current Flutter sample:https://github.com/Esri/arcgis-maps-sdk-flutter-samples/tree/v.next/lib/samples/display_map_from_mobile_map_package
 

Updates the sample to match the common design:
- Added comments in the code
- Added the main README
- Generated the README metadata

### CODE REVIEW

File: lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart [hunk 1]
```diff
@@ -32,10 +32,12 @@ class DisplayMapFromMobileMapPackageSample extends StatefulWidget {
...
 class _DisplayMapFromMobileMapPackageSampleState
     extends State<DisplayMapFromMobileMapPackageSample> {
+  // creates a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
 
   @override
   Widget build(BuildContext context) {
+    // creates a map view to display the map.
     return Scaffold(
       body: ArcGISMapView(
```
Comment (jenmerritt) [inline lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart:42]:
```suggestion
    return Scaffold(
      // add a map view to the widget tree and set a controller.
      body: ArcGISMapView(
```

File: lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart [hunk 2]
```diff
@@ -47,9 +49,13 @@ class _DisplayMapFromMobileMapPackageSampleState
...
+
+    // loads the local mobile map package.
     final mmpkFile = File('${appDir.absolute.path}/Yellowstone.mmpk');
     final mmpk = MobileMapPackage.withFileUri(mmpkFile.uri);
     await mmpk.load();
+
+    // gets the first map in the mobile map package.
     if (mmpk.maps.isNotEmpty) {
       _mapViewController.arcGISMap = mmpk.maps.first;
     }
```
Comment (jenmerritt) [inline lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart:61]:
```suggestion
    if (mmpk.maps.isNotEmpty) {
        // get the first map in the mobile map package and set to the map view.
      _mapViewController.arcGISMap = mmpk.maps.first;
    }
```

File: lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart [hunk 3]
```diff
@@ -47,9 +49,13 @@ class _DisplayMapFromMobileMapPackageSampleState
   void onMapViewReady() async {
     await downloadSampleData(['e1f3a7254cb845b09450f54937c16061']);
     final appDir = await getApplicationDocumentsDirectory();
+
+    // loads the local mobile map package.
```
Comment (jenmerritt) [inline lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart:53]:
```suggestion
    // load the local mobile map package.
```

File: lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart [hunk 4]
```diff
@@ -32,10 +32,12 @@ class DisplayMapFromMobileMapPackageSample extends StatefulWidget {
 
 class _DisplayMapFromMobileMapPackageSampleState
     extends State<DisplayMapFromMobileMapPackageSample> {
+  // creates a controller for the map view.
```
Comment (jenmerritt) [inline lib/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart:35]:
```suggestion
  // create a controller for the map view.
```
These comments are good - just make the verb more like "do this" than "this does this" so it reads like instructions, if that makes sense.

**Review by jenmerritt — COMMENTED** [review]
This sample looks good @shokhinaToj , just a few notes on the comments.

**Review by PaulAllanSturm — APPROVED** [review]

**Review by jenmerritt — APPROVED** [review]
👍

**Pull request merged**